packages/blade/src/components/Button/BaseButton/StyledBaseButton.d.ts

```diff
@@ -1,9 +1,13 @@
-export type StyledBaseButtonProps = {
+import type { BaseButtonProps } from './BaseButton';
+
+export type StyledBaseButtonProps = Omit<BaseButtonProps, 'icon' | 'children'> & {
   children: string;
   onClick: () => void;
   color: string;
   hoverColor: string;
   activeColor: string;
+  buttonHeight: '48px' | '40px' | '32px' | '28px';
```

Comment:
put it under button tokens?

Reply:
Will do this in the #487 PR. Would cause a lot of rebasing headache otherwise.

Reply:
Done in #487